verification/VerifyTasks.wdl

```diff
@@ -159,15 +160,37 @@ task CompareBams {
           ~{test_bam} \
           ~{truth_bam} \
           O=comparison.tsv \
-          LENIENT_HEADER=true
-
+          ~{true='LENIENT_HEADER=true' false="LENIENT_HEADER=false" lenient_header}
```

Comment:
Can you do `LENIENT_HEADER=~{lenient_header}` or does that fail since `lenient_header` is a boolean rather than a string?

Reply:
Not sure.  I kind of doubt it, but will try.

Reply:
But it did work!  Thanks.